controllers/imageclusterinstall_controller.go

```diff
@@ -186,49 +202,72 @@ func (r *ClusterConfigReconciler) setHostConfiguredCondition(ctx context.Context
 		cond.Message = err.Error()
 	}
 
-	patch := client.MergeFrom(config.DeepCopy())
-	meta.SetStatusCondition(&config.Status.Conditions, cond)
-	return r.Status().Patch(ctx, config, patch)
+	patch := client.MergeFrom(clusterInstall.DeepCopy())
+	meta.SetStatusCondition(&clusterInstall.Status.ConfigConditions, cond)
+	return r.Status().Patch(ctx, clusterInstall, patch)
 }
 
-func (r *ClusterConfigReconciler) mapBMHToCC(ctx context.Context, obj client.Object) []reconcile.Request {
+func (r *ImageClusterInstallReconciler) mapBMHToICI(ctx context.Context, obj client.Object) []reconcile.Request {
 	bmh := &bmh_v1alpha1.BareMetalHost{}
 	bmhName := obj.GetName()
 	bmhNamespace := obj.GetNamespace()
 
 	if err := r.Get(ctx, types.NamespacedName{Name: bmhName, Namespace: bmhNamespace}, bmh); err != nil {
 		return []reconcile.Request{}
 	}
-	ccList := &relocationv1alpha1.ClusterConfigList{}
-	if err := r.List(ctx, ccList); err != nil {
+	iciList := &relocationv1alpha1.ImageClusterInstallList{}
+	if err := r.List(ctx, iciList); err != nil {
 		return []reconcile.Request{}
 	}
-	if len(ccList.Items) == 0 {
+	if len(iciList.Items) == 0 {
 		return []reconcile.Request{}
 	}
 
 	requests := []reconcile.Request{}
-	for _, cc := range ccList.Items {
-		if cc.Spec.BareMetalHostRef == nil {
+	for _, ici := range iciList.Items {
+		if ici.Spec.BareMetalHostRef == nil {
 			continue
 		}
-		if cc.Spec.BareMetalHostRef.Name == bmhName && cc.Spec.BareMetalHostRef.Namespace == bmhNamespace {
+		if ici.Spec.BareMetalHostRef.Name == bmhName && ici.Spec.BareMetalHostRef.Namespace == bmhNamespace {
 			req := reconcile.Request{
 				NamespacedName: types.NamespacedName{
-					Namespace: cc.Namespace,
-					Name:      cc.Name,
+					Namespace: ici.Namespace,
+					Name:      ici.Name,
 				},
 			}
 			requests = append(requests, req)
 		}
 	}
 	if len(requests) > 1 {
-		r.Log.Warn("found multiple ClusterConfigs referencing BaremetalHost %s/%s", bmhNamespace, bmhName)
+		r.Log.Warn("found multiple ImageClusterInstalls referencing BaremetalHost %s/%s", bmhNamespace, bmhName)
 	}
 	return requests
 }
 
-func serviceURL(opts *ClusterConfigReconcilerOptions) string {
+func (r *ImageClusterInstallReconciler) mapCDToICI(ctx context.Context, obj client.Object) []reconcile.Request {
+	cdName := obj.GetName()
+	cdNamespace := obj.GetNamespace()
+
+	cd := &hivev1.ClusterDeployment{}
+	if err := r.Get(ctx, types.NamespacedName{Name: cdName, Namespace: cdNamespace}, cd); err != nil {
+		return []reconcile.Request{}
+	}
+
+	if cd.Spec.ClusterInstallRef != nil &&
+		cd.Spec.ClusterInstallRef.Group == relocationv1alpha1.Group &&
```

Comment:
`relocationv1alpha1`?

Reply:
That's the group name for this operator.

I figure we can change it once we decide on the new repo name.

Reply:
Let's change it once the repo name is updated

Reply:
Do you want to change this in this PR now that the name is changed or do it in a follow up? My vote is a follow up FWIW

Reply:
In a followup